package.json

```diff
@@ -58,41 +58,42 @@
   "homepage": "https://mobx.js.org/",
   "dependencies": {},
   "devDependencies": {
-    "@babel/core": "^7.3.4",
-    "@babel/plugin-proposal-class-properties": "^7.3.4",
-    "@babel/plugin-proposal-decorators": "^7.3.0",
-    "@babel/plugin-transform-runtime": "^7.3.4",
-    "@babel/preset-env": "^7.3.4",
-    "@babel/runtime": "^7.3.4",
-    "@size-limit/preset-big-lib": "^3.0.0",
+    "@babel/core": "^7.8.4",
+    "@babel/plugin-proposal-class-properties": "^7.8.3",
+    "@babel/plugin-proposal-decorators": "^7.8.3",
+    "@babel/plugin-transform-runtime": "^7.8.3",
+    "@babel/preset-env": "^7.8.4",
+    "@babel/runtime": "^7.8.4",
+    "@rollup/plugin-node-resolve": "^7.1.1",
+    "@size-limit/preset-big-lib": "^4.0.2",
     "@types/jest": "^24.0.11",
-    "@types/node": "^11.11.3",
-    "@typescript-eslint/eslint-plugin": "^1.4.2",
+    "@types/node": "^12",
+    "@typescript-eslint/eslint-plugin": "^2.20.0",
+    "@typescript-eslint/parser": "^2.20.0",
     "babel-jest": "^24.5.0",
-    "chalk": "^1.1.3",
     "conditional-type-checks": "^1.0.4",
     "coveralls": "^3.0.3",
-    "eslint": "^5.15.2",
+    "eslint": "^6.8.0",
     "flow-bin": "^0.59.0",
-    "fs-extra": "^7.0.1",
-    "husky": "^1.3.1",
+    "fs-extra": "^8.1.0",
+    "husky": "^4.2.3",
     "iterall": "^1.2.2",
     "jest": "^24.5.0",
-    "prettier": "^1.16.4",
+    "prettier": "^1.19.1",
     "pretty-quick": "2.0.1",
     "prompts": "^2.3.0",
-    "rollup": "^1.6.0",
-    "rollup-plugin-filesize": "^6.0.1",
-    "rollup-plugin-node-resolve": "^4.0.1",
+    "rollup": "^1.31.1",
+    "rollup-plugin-filesize": "^6.2.1",
     "rollup-plugin-replace": "^2.1.0",
-    "rollup-plugin-terser": "^4.0.4",
-    "semver": "^7.1.1",
+    "rollup-plugin-terser": "^5.2.0",
+    "semver": "^7.1.3",
     "serializr": "^1.5.1",
     "shelljs": "^0.8.3",
+    "size-limit": "^4.0.2",
```

Comment:
Not a big deal, but the `@size-limit/preset-big-lib` has it as dependency already and actually warns about it when you run it.

Reply:
it was moved to peerDeps there

Reply:
https://github.com/ai/size-limit/blob/master/packages/preset-big-lib/package.json#L15-L23